tenderly-suite/role-update-action/role-update/roleUpdate.js

```diff
@@ -0,0 +1,146 @@
+const { ethers } = require('ethers');
+const {
+    utils: { toUtf8Bytes },
+} = ethers;
+const axios = require('axios').default;
+
+const PAGER_DUTY_ALERT_URL = 'https://events.pagerduty.com/v2/enqueue';
+
+const handleRoleUpdate = async (context, event) => {
+    if (!event || !event.logs || !context || !context.metadata) {
+        throw new Error('INVALID_INPUT_FOR_ACTION');
+    }
+
+    const { rolesAdded, rolesRemoved } = await context.storage.getJson('EventsABI');
+    const rolesAddedHash = ethers.utils.keccak256(toUtf8Bytes(rolesAdded));
+    const rolesRemovedHash = ethers.utils.keccak256(toUtf8Bytes(rolesRemoved));
+
+    const chainName = context.metadata.getNetwork();
+    const trustedAddresses = await context.storage.getJson('TrustedAddresses');
+
+    const roleAddedAccounts = [];
+    const addedRoles = [];
+    const roleRemovedAccounts = [];
+    const removedRoles = [];
+    const summary = 'Roles updated';
+    let severity = 0;
+
+    for (const log of event.logs) {
+        if (log.topics[0] === rolesAddedHash || log.topics[0] === rolesRemovedHash) {
+            const isRoleAdded = log.topics[0] === rolesAddedHash;
+
+            if (log.data.length <= 2) {
+                throw new Error('EMPTY_LOG_DATA');
+            }
+
+            const roles = toRoleArray(parseInt(log.data, 16));
+
+            if (log.topics.length === 0) {
+                throw new Error('INVALID_LOGS_LENGTH');
+            }
+
+            if (log.topics.length === 0) {
+                throw new Error('INVALID_LOG_TOPICS_LENGTH');
+            }
+
+            if (log.topics[1].length < 26 + 40) {
+                throw new Error('INVALID_LOG_TOPIC_LENGTH');
+            }
```

Comment:
```suggestion
            if (log.topics[1].length !== 66) {
                throw new Error('INVALID_LOG_TOPIC_LENGTH');
            }
```
Log topic length should always be 66 (including '0x') right?